packs/campaign-effects/resonant-reflection-reflection-of-life.json

```diff
@@ -30,6 +30,53 @@
                 "mode": "add",
                 "path": "system.attributes.hp.recoveryMultiplier",
                 "value": 1
+            },
+            {
+                "itemType": "action",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "item:slug:activate-resonant-reflection",
+                    "resonant:reflection-of-life"
+                ],
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.ResonantReflection.ReflectionOfLife.Description",
+                        "title": "PF2E.SpecificRule.ResonantReflection.ReflectionOfLife.Title"
+                    }
+                ]
+            },
+            {
+                "alwaysActive": true,
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.ResonantReflection.Label",
+                "mergeable": true,
+                "option": "resonant",
+                "placement": "actions",
+                "suboptions": [
+                    {
+                        "label": "{item|name}",
+                        "value": "reflection-of-life"
+                    }
+                ],
+                "toggleable": true
+            },
+            {
+                "key": "RollOption",
+                "domain": "all",
+                "option": "resonant-reflection"
+            },
```

Comment:
The reason this is failing is that you have this RE on the resonant reflections. Put it on the activation action.

Reply:
Done. Check when you get a moment. 

Thanks :)